wrapper/src/test/java/integration/container/tests/TopologyQueryTests.java

```diff
@@ -0,0 +1,273 @@
+import integration.container.TestEnvironment;
+import integration.container.condition.DisableOnTestFeature;
+import integration.container.condition.EnableOnDatabaseEngineDeployment;
+import integration.container.condition.EnableOnNumOfInstances;
+import integration.util.AuroraTestUtility;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Properties;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.MethodOrderer.MethodName;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.jdbc.dialect.AuroraMysqlDialect;
+import software.amazon.jdbc.dialect.AuroraPgDialect;
+import software.amazon.jdbc.dialect.RdsMultiAzDbClusterMysqlDialect;
+import software.amazon.jdbc.dialect.RdsMultiAzDbClusterPgDialect;
+
+@TestMethodOrder(MethodName.class)
+@DisableOnTestFeature({
+    TestEnvironmentFeatures.PERFORMANCE,
+    TestEnvironmentFeatures.RUN_HIBERNATE_TESTS_ONLY,
+    TestEnvironmentFeatures.RUN_AUTOSCALING_TESTS_ONLY})
+public class TopologyQueryTests {
+  private static final Logger LOGGER = Logger.getLogger(TopologyQueryTests.class.getName());
+  private final AuroraTestUtility util = new AuroraTestUtility();
+
+  @TestTemplate
+  @EnableOnDatabaseEngineDeployment(DatabaseEngineDeployment.AURORA)
+  @ExtendWith(TestDriverProvider.class)
+  public void testAuroraTypes(TestDriver testDriver) throws SQLException {
+    LOGGER.info(testDriver.toString());
+    List<String> expectedTypes;
+
+    final Properties props = ConnectionStringHelper.getDefaultPropertiesWithNoPlugins();
+    DriverHelper.setConnectTimeout(testDriver, props, 10, TimeUnit.SECONDS);
+    DriverHelper.setSocketTimeout(testDriver, props, 10, TimeUnit.SECONDS);
+
+    String url =
+        ConnectionStringHelper.getWrapperUrl(
+            testDriver,
+            TestEnvironment.getCurrent()
+                .getInfo()
+                .getDatabaseInfo()
+                .getInstances()
+                .get(0)
+                .getHost(),
+            TestEnvironment.getCurrent()
+                .getInfo()
+                .getDatabaseInfo()
+                .getInstances()
+                .get(0)
+                .getPort(),
+            TestEnvironment.getCurrent().getInfo().getDatabaseInfo().getDefaultDbName());
+    LOGGER.finest("Connecting to " + url);
+
+    String query = null;
+    if (TestEnvironment.getCurrent().getCurrentDriver() == TestDriver.PG) {
```

Comment:
What's about MariaDb? Is it supported by the test?

Reply:
It's not

Reply:
I mean I don't see a branch of the code that handle the case with mariaDb. I see AuroraMysql and AuroraPG.